cypress/integration/portal-ui.spec.js

```diff
@@ -1,12 +1,46 @@
 describe('portal-ui', () => {
-  beforeEach(() => {
-    // Any request we do not explicitly route will return 404,
-    // so we won't end up depending on outside resources by accident.
-    cy.server({ force404: true });
+  it('has nice 404', () => {
+    // Homepage
+    cy.visit('/no-such-page', {failOnStatusCode: false});
+    cy.contains('404: Not Found');
+    cy.contains('If this page should exist');
   });
 
   it('works', () => {
```

Comment:
for the description, 'perhaps correctly loads hubmap landing page'.

Reply:
Scope of the test has expanded, but will retitle.